modules/extension/xsd/xsd-gml3/src/main/java/org/geotools/gml3/GMLSchema.java

```diff
@@ -11319,7 +11319,7 @@ private static AttributeType build_MULTISURFACETYPE_TYPE() {
         AttributeType builtType =
                 new AttributeTypeImpl(
                         new NameImpl("http://www.opengis.net/gml", "MultiSurfaceType"),
-                        org.locationtech.jts.geom.MultiPolygon.class,
+                        MultiSurface.class,
```

Comment:
These GML schema changes must be covered by some type of test (I presume it's going to be a parsing/encoding type test). Same goes for the GML 3.2 change

Reply:
Still needs a test. While you're at it, I'd recommend rebasing on the main branch, big changes have occurrend in the meantime.

Reply:
Yes !! merge is OK now